src/Dialect/ONNX/IndexExpr.cpp

```diff
@@ -295,6 +296,25 @@ void IndexExprContext::createStoreOp(
   }
 }
 
+//===----------------------------------------------------------------------===//
+// IndexExprContext support for creating krnl load and store ops.
+//===----------------------------------------------------------------------===//
+Value IndexExprContext::createKrnlLoadOp(
+    Value memref, SmallVectorImpl<IndexExpr> &indices) {
+  SmallVector<Value, 4> loadIndices;
+  for (IndexExpr ie : indices)
+    loadIndices.emplace_back(ie.getValue());
+  return getRewriter().create<KrnlLoadOp>(getLoc(), memref, loadIndices);
+}
+
+void IndexExprContext::createKrnlStoreOp(
+    Value val, Value memref, SmallVectorImpl<IndexExpr> &indices) {
+  SmallVector<Value, 4> storeIndices;
+  for (IndexExpr ie : indices)
+    storeIndices.emplace_back(ie.getValue());
+  getRewriter().create<KrnlStoreOp>(getLoc(), val, memref, storeIndices);
+}
+
```

Comment:
I have seen other occurrence of create<AffineLoadOp> in IndexExpr.cpp, can you double check that they are all converted to the krnl versions?

Reply:
Oops, thanks for pointing it out! Fixed them.